packages/sources/src/styling/global/ul.scss

```diff
@@ -0,0 +1,17 @@
+ul.dso-columns-list {
```

Comment:
De files in `sources/src/styling/global` zijn voor basis styling, nu heb je er een (sub) component geplaatst.

En waarom is deze zo hard op `ul` gescoped?

Reply:
Als ze iets (alfabetisch) sorteren willen we ze toch de ruimte geven voor 'n `ol`..

Reply:
Afgesproken is: We doen vast een voorzet op het toekomstige HTML/CSS Component `List`, en maken hier in de `/dso-toolkit/` package een `list.scss` waar deze stijling naar toe verhuist. Met als toevoeging ook de `ol`